STM32F303K6Tx/Core/Src/main.c

```diff
@@ -62,7 +65,8 @@ uint32_t sum = 0;
 uint16_t mean = 0;
 float offset[4] = {-31.25, -31.25, 0, 0};
 uint8_t bytes[4];
```

Comment:
I don't think we need this var anymore since we are no longer using float to bytes